conf/websocket.conf

```diff
@@ -39,6 +39,10 @@ authenticationProviders=
 # Enforce authorization
 authorizationEnabled=false
 
+# Role names that are treated as "super-user", meaning they will be able to do all admin
+# operations and publish/consume from all topics
+superUserRoles=
+
```

Comment:
Actually, I didn't add it into config to avoid confusion as ```serviceUrl ``` is optional if ```globalZookeeperServers``` is already present. 

We have still kept ```clusterName``` in config because it is required by [Authorization](https://github.com/yahoo/pulsar/blob/master/pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/authorization/AuthorizationManager.java#L108), and proxy [fetches](https://github.com/rdhabalia/pulsar/blob/a0d7007123301016124ff887d395d11a669335b5/pulsar-websocket/src/main/java/com/yahoo/pulsar/websocket/WebSocketService.java#L152) ```serviceUrl``` from ```zookeeper``` if ```globalZookeeperServers``` and ```clusterName``` is present.

Do you think, we should still keep ```serviceUrl``` by providing comment with short description as mentioned above?